packages/forma-36-website/src/components/ComponentSource.js

```diff
@@ -3,7 +3,15 @@ import { css } from '@emotion/core';
 import tokens from '@contentful/forma-36-tokens';
 import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
 import * as f36Components from '@contentful/forma-36-react-components';
-import { Grid, GridItem, Autocomplete } from '@contentful/forma-36-react-components/dist/alpha';
+import {
+  Grid,
+  GridItem,
+  Autocomplete,
+  Flex,
+  ProductIcon,
+  Accordion,
+  AccordionItem,
+} from '@contentful/forma-36-react-components/dist/alpha';
```

Comment:
Wdyt about doing the same as the line above?

```js
import * as f36Components from '@contentful/forma-36-react-components';
import * as f36AlphaComponents from '@contentful/forma-36-react-components/dist/alpha';
```

Reply:
🤦‍♀️✅